Package.swift

```diff
@@ -13,26 +13,34 @@ let package = Package(
             targets: ["MapboxMaps"]),
     ],
     dependencies: [
-        .package(name: "MapboxCoreMaps", url: "https://github.com/mapbox/mapbox-core-maps-ios.git", .exact("10.5.1")),
-        .package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("21.3.0")),
-        .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", .exact("1.0.7")),
-        .package(name: "Turf", url: "https://github.com/mapbox/turf-swift.git", from: "2.0.0"),
-        .package(name: "CocoaImageHashing", url: "https://github.com/ameingast/cocoaimagehashing", .exact("1.9.0"))
+        .package(url: "https://github.com/mapbox/mapbox-core-maps-ios.git", revision: "v10.5.1"),
+        .package(url: "https://github.com/mapbox/mapbox-common-ios.git", revision: "v21.3.0"),
+        .package(url: "https://github.com/mapbox/mapbox-events-ios.git", revision: "v1.0.7"),
+        .package(url: "https://github.com/mapbox/turf-swift.git", from: "2.0.0"),
+        .package(url: "https://github.com/ameingast/cocoaimagehashing", from: "1.9.0"),
```

Comment:
Is it a required change for swift-tools 5.5? 
The previous version looks a bit better for me as we name the product just one in dependencies rather than several times in different places. I suppose we don't have a multi-product packages?

Reply:
The old methods are now deprecated and will be removed in a future version.